core/web/jobs_controller.go

```diff
@@ -47,8 +47,8 @@ func (jc *JobsController) Index(c *gin.Context, size, page, offset int) {
 		return
 	}
 	var resources []presenters.JobResource
-	for _, job := range jobs {
-		resources = append(resources, *presenters.NewJobResource(job))
+	for _, individualJob := range jobs {
```

Comment:
nit, but I don't see this as really adding much?

Reply:
There was a warning from the IDE complaining that the variable name is the same as the package name (the job package). Happy to remove it if you think it's not a warning worth considering.

Reply:
Any final thoughts on this @RyanRHall?

Reply:
okay, I'm fine with it